README.md

```diff
@@ -18,6 +18,7 @@ We have created this site with this mission in mind, to share and discuss open-s
 | [Open data standards](https://github.com/NHSDigital/open-data-standards) | Draft of open data CSV standards for comment | Markdown | No |
 | [Data Visualisation for Health and Care Community of Practice](https://github.com/NHSDigital/data-viz-community-of-practice) | Committe of folk working in data visualisation across CSUs, Trusts, NHS Digital, NHS England and others. | Various | No |
 | [Medicines text mining tool](https://github.com/NHSDigital/medicines-text-mining-tool) | Medicines text mining application. | Python, PySpark | No |
+| [MPS Diagnostics](https://github.com/NHSDigital/mps_diagnostics) | Interpretable metadata for MPS record linkage and Person ID | Python, PySpark | No |
```

Comment:
looks great - though I guess you have published a report of some sort? https://github.com/NHSDigital/mps_diagnostics/blob/main/MPS%20Diagnostics%20User%20Guide%201.0.0.pdf